client/src/components/Home/Home.jsx

```diff
@@ -1,9 +1,142 @@
 import React from 'react';
-import { Container, Header } from './Home.styled';
+import {
+	Container,
```

Comment:
This is a _lot_ of components 😄 	Maybe restructure Home page into a few different components do avoid code clutter?

Reply:
Definitely a good idea, it was getting unruly 😅